addon/list-utils.js

```diff
@@ -133,7 +133,9 @@ export function getItemsFromAjaxCall ({ajax, bunsenId, data, filter, options, va
  */
 export function getItemsFromEmberData (value, modelDef, data, bunsenId, store, filter) {
   const modelType = modelDef.modelType || 'resources'
-  const {labelAttribute, valueAttribute} = modelDef
+  const {labelAttribute, queryForCurrentValue, valueAttribute} = modelDef
+  const valueAsId = Number(value[bunsenId])
+  const actuallyFindCurrentValue = queryForCurrentValue && !!valueAsId
```

Comment:
What if someone has a number value that isn't an id? 

And if you don't have a paged API, why not avoid the extra query?